includes/ajax.php

```diff
@@ -479,8 +479,26 @@ function tsml_ajax_geocode()
         }
 
         //add custom meeting fields if available
-        foreach (['types', 'data_source', 'conference_url', 'conference_url_notes', 'conference_phone', 'conference_phone_notes'] as $key) {
-            if (!empty($meeting[$key])) add_post_meta($meeting_id, $key, $meeting[$key]);
+       foreach (['types', 'data_source', 'conference_url', 'conference_url_notes', 'conference_phone', 'conference_phone_notes'] as $key) {
+           if (!empty($meeting[$key])) {
+               add_post_meta($meeting_id, $key, $meeting[$key]);
+           }
+       }
+
+        //import custom user-defined meeting fields from CSV 
+        if (!empty($tsml_custom_meeting_fields)) {
+            foreach ($tsml_custom_meeting_fields as $field => $title) {
+                if (array_key_exists($field, $meeting) && !empty($meeting[$field])) {
+                    $import_id = (int) $meeting['id'];
+                    $meta_value = (string) $meeting[$field];
+                    add_post_meta($import_id, $field, $meta_value);
```

Comment:
i guess i'm a little confused by the difference between $meeting['id'] and $meeting_id - the former comes from the buffer and the latter is what we just created? 

could you instead make this `add_post_meta($meeting_id, $field, $meta_value);`? that way you might not need the `$import_id` variable or the SQL query below

Reply:
That's what I originally had, but no joy. I spent numerous hours trying to figure a work around. The import_id is just for clarity.

Reply:
add_post_meta($meeting_id, $field, $meta_value);  coded and tested...